bpf/bpf_host.c

```diff
@@ -902,12 +902,20 @@ handle_netdev(struct __ctx_buff *ctx, const bool from_host)
 	return do_netdev(ctx, proto, from_host);
 }
 
+/*
+ * from-netdev is attached as a tc egress filter
```

Comment:
The `from-netdev` section is attached to a native device (configured via `--devices` or auto-detected by `daemon/cmd/kubeproxy_replacement.go`; same devices as in the `to-netdev` case) tc ingress.

Reply:
Thanks, I have no idea how/why I missed that..